presentation-api/controlling-ua/reconnectToPresentation_error-manual.html

```diff
@@ -0,0 +1,43 @@
+<!doctype html>
+<meta charset="utf-8">
+<title>Presentation API reconnect a presentation for Controlling User Agent (Error - manual test)</title>
+<link rel="author" title="Franck William Taffo" href="http://www.fokus.fraunhofer.de">
+<link rel="help" href="http://w3c.github.io/presentation-api/#dfn-controlling-user-agent">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+
+    /**
+     *
+     * Test if reconnect returns a NotFoundError() by wrong presentation ID
+     */
+
+    var client_id = String(new Date().getTime()) + String(Math.floor(Math.random() * 1e5));
+    //relative presentation URL
+    var presentation_url = "../receiving-ua/idlharness.html#__castAppId__=2334D33A/__castClientId__="+ client_id;
+    var request = new PresentationRequest(presentation_url);
+    var wrong_presentationId = null;
+
+    var reconnect = function () {
+        promise_test(function () {
+
+            var presId = "presId";
+            // presId is mandatory when reconnecting to a presentation.
+            if (!!presId) {
```

Comment:
For this case, you don't need to do this check since it is just assigned a string.